x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/use_columns.tsx

```diff
@@ -195,17 +195,53 @@ const useActionsColumn = (): EuiTableActionsColumnType<Rule> => {
 };
 
 export const useRulesColumns = ({ hasPermissions }: ColumnsProps): TableColumn[] => {
+  const { triggersActionsUi } = useKibana().services;
   const actionsColumn = useActionsColumn();
   const enabledColumn = useEnabledColumn({ hasPermissions });
   const ruleNameColumn = useRuleNameColumn();
-  const { isInMemorySorting } = useRulesTableContext().state;
+  const {
+    state: { isInMemorySorting, loadingRuleIds, isLoading },
+    actions: { reFetchRules },
+  } = useRulesTableContext();
   const [showRelatedIntegrations] = useUiSetting$<boolean>(SHOW_RELATED_INTEGRATIONS_SETTING);
 
   return useMemo(
     () => [
       ruleNameColumn,
       ...(showRelatedIntegrations ? [INTEGRATIONS_COLUMN] : []),
       TAGS_COLUMN,
+      {
+        id: 'ruleSnoozeNotify',
+        name: (
+          <EuiToolTip
+            data-test-subj="rulesTableCell-notifyTooltip"
+            content={i18n.COLUMN_NOTIFY_TOOLTIP}
+          >
+            <span>
+              {i18n.COLUMN_NOTIFY}
+              &nbsp;
+              <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+            </span>
+          </EuiToolTip>
+        ),
+        width: '14%',
+        'data-test-subj': 'rulesTableCell-rulesListNotify',
+        render: (rule: Rule) => {
+          return triggersActionsUi.getRulesListNotifyBadge({
+            rule: {
+              ...rule,
```

Comment:
Why do you need the whole rule here? Let's limit it to the only properties you actually need, e.g. `id`.